crates/rattler_conda_types/src/prefix_record.rs

```diff
@@ -85,6 +87,14 @@ pub struct PathsEntry {
     /// The size of the file in bytes
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub size_in_bytes: Option<u64>,
+
+    /// The file mode of the entry. This is used in conjunction with a prefix_placeholder
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub file_mode: Option<FileMode>,
+
+    /// The original sentinel value used for prefix-replacement from the package
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub prefix_placeholder: Option<String>,
```

Comment:
Is it still useful to have this? This will have been replaced once the files have been installed.

Reply:
I guess what I dont understand is why the original placeholder is needed? This has already been replaced with a predictable value, namely the prefix that the files are in? It seems redundant to store the original placeholder in that case.

Reply:
👍